frontend/static/js/charts/fees-feerate-bands-stacked.js

```diff
@@ -0,0 +1,126 @@
+const ANNOTATIONS = [annotationInscriptionsHype, annotationRunestones]
+const MOVING_AVERAGE_DAYS = MOVING_AVERAGE_7D
+const NAME = "fee rate distribution by bands"
+const PRECISION = 1
+let START_DATE = new Date();
+START_DATE.setMonth(new Date().getMonth() - 6); // Show last 6 months for thumbnail
```

Comment:
I think it would be nice to show a bit more historical data by default. Maybe even 5 years.